VBAProjectFiles/IOFunctions.bas

```diff
@@ -138,36 +76,6 @@ Function CheckErrFile(filePath As String) As String
     CheckErrFile = ""
 End Function
 
-Function ConvertESOFile(filePath As String) As Boolean
-    AssembleConstList
-    
-    Dim rviFile As String
-    rviFile = GetOutputFolder() + "\Rvi.rvi"
```

Comment:
Did we remove this functionality completely? This rvi file is used to filter the CSV output.